physics/horizontal_projectile_motion.py

```diff
@@ -0,0 +1,131 @@
+"""
+Horizontal Projectile Motion problem in physics.
+This algorithm solves a specific problem in which
+the motion starts from the ground as can be seen below:
+      (v = 0)
+        **
+       *  *
+      *    *
+     *      *
+    *        *
+   *          *
+GROUND      GROUND
+
+For more info: https://en.wikipedia.org/wiki/Projectile_motion
+"""
+
+# Importing packages
+from math import pi, sin
+
+# Acceleration Constant on hearth (unit m/s^2)
+g = 9.80665
+
+
+def angle_to_radians(angle : float) -> float:
+    """
+    Convert an angle from degrees to randians
+    """
```

Comment:
```python
from math import radians
```
or 
```python
from math import radians as angle_to_radians
```
use radians instead of `angle_to_radians`.

Reply:
Thank you!

 I did change it to the second option you suggested.